src/main/java/Event.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Event Class.
+ * Event Class is a subclass of Task Class. It stores the description of this instance
+ * as well as the date and time of the particular event.
+ *
+ * @author Amos Cheong
+ */
+public class Event extends Task {
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+    /**
+     * Get the description of this Event object.
```

Comment:
Same comment that I had for one of the previous javadoc comments - you can leave a blank line between the description and the return thing. Also, you don't need the javadoc comments for small, self-explanatory functions.